website/addons/figshare/messages.py

```diff
@@ -1,9 +1,9 @@
 FIGSHARE = 'figshare'
 
 # MODEL MESSAGES :model.py
-BEFORE_PAGE_LOAD_PRIVATE_NODE_MIXED_FS = 'Warning: This OSF {category} is private but ' + FIGSHARE + ' project {project_id} may contain some public files or filesets. '
+BEFORE_PAGE_LOAD_PRIVATE_NODE_MIXED_FS = 'Warning: This OSF {category} is private but ' + FIGSHARE + ' project {project_id} may contain some public files or filesets. The files in this Figshare project can be viewed <a href="https://http://figshare.com/articles/{project_id}/{figshare_id}">here</a>'
```

Comment:
Actually the example in the issue specifically addresses a message triggered by messages.BEFORE_PAGE_LOAD_PRIVATE_NODE_MIXED_FS